### ISSUE TITLE
The option parameter does not affect the VAD process.

### ISSUE THREAD

**Issue by zh-plus** [SYSTRAN/faster-whisper#916]
After #856, the VAD-related `option` paramter does not affect the vad results at all.

https://github.com/SYSTRAN/faster-whisper/blob/eb8390233c160a8232abf88f9b949eb5cbc48df8/faster_whisper/transcribe.py#L120

I believe `self.options` isn't used during VAD. Should we modify `VadOptions`， and make it take effect here?

https://github.com/SYSTRAN/faster-whisper/blob/eb8390233c160a8232abf88f9b949eb5cbc48df8/faster_whisper/transcribe.py#L381

**Comment by Jiltseb** [comment]
I think you are confused between `VadOptions` (which is a silero VAD related parameter) and `options` variable (that is used to initialize the inference pipeline with default transcription options). 

Batched version of faster whisper does not use silero VAD at the moment, hence the `VadOptions` variable is not needed for VAD. 
If required, it is possible to make the `vad_onset` and `vad_offset` user configurable for tuning the VAD model in batched inference.

**Comment by zh-plus** [comment]
Thank you for your response! I just noticed the difference.

I think it would be beneficial to make vad_onset and vad_offset user-configurable. This flexibility would allow users to adjust these parameters according to their specific data requirements.

If needed, I may take a PR for this issue soon.
